runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -39,3 +51,43 @@ J9::ARM64::TreeEvaluator::generateFillInDataBlockSequenceForUnresolvedField(TR::
    TR_ASSERT_FATAL(false, "This helper implements platform specific code for Fieldwatch, which is currently not supported on ARM64 platforms.\n");
    }
 	
+TR::Register *J9::ARM64::TreeEvaluator::DIVCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Node *divisor = node->getFirstChild()->getSecondChild();
+   TR::Register *srcReg = NULL;
```

Comment:
I'd prefer `divisorReg` for readability.  Also, please sink it into the `if` closer to where it is used.